ocs_ci/templates/CSI/addons/network-fence-rbd.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: csiaddons.openshift.io/v1alpha1
+kind: NetworkFence
+metadata:
+  name: <openshift-node-name>
+spec:
+  cidrs:
+  - <ip/32>
+  driver: openshift-storage.rbd.csi.ceph.com
+  fenceState: Fenced
```

Comment:
is this by default value ?or we are suppose to add  while executing tc?
I am just relatingthis with DR. we add this line while doing fencing.

Reply:
No this is just default value. As the first time we create it is meant to be fenced.